services/tacoService.js

```diff
@@ -0,0 +1,32 @@
+const Taco = require('../models/Taco');
+
+const createTaco = (recipientUserId, giverUserId, number, reason) => {
+    let taco = new Taco();
+    taco.id = taco._id;
+    taco.recipientUserId = recipientUserId;
+    taco.giverUserId = giverUserId;
+    taco.number = number;
+    taco.reason = reason;
+    taco.save((err) => {
+        if(err) { console.error(err)}
+    });
+};
+
+const getNumberOfGiftedTacosByUserInLastDay = (userid) => {
```

Comment:
pretty sure this doesn't return a number and you seem to be using the response as a number. Is especially doesn't return the total number of tacos because one taco document can store multiple tacos. I would suggest creating a new taco document for every taco to make counting easier

Reply:
I don't like the idea of storing individual tacos in their own documents. I was thinking of this more as a taco transaction document. I did forget about adding that num tacos though, and will do some maths on it.